bugbug/nlp.py

```diff
@@ -3,17 +3,38 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from __future__ import print_function
+
+import sys
 from collections import defaultdict
 from functools import lru_cache
 
 import numpy as np
-import spacy
-from gensim.models import KeyedVectors
 from sklearn.base import BaseEstimator, TransformerMixin
 from sklearn.feature_extraction.text import TfidfVectorizer
-from spacy.tokenizer import Tokenizer
 
-nlp = spacy.load("en_core_web_sm")
+# Spacy is an optional dependency
+try:
+    import spacy
+
+    from spacy.tokenizer import Tokenizer
+    from gensim.models import KeyedVectors
```

Comment:
We should also detect when gensim is not available.

Reply:
Indeed, I introduced a new atomic boolean for this effect